PowerEditor/src/MISC/Common/Common.cpp

```diff
@@ -1137,6 +1137,20 @@ HWND CreateToolTipRect(int toolID, HWND hWnd, HINSTANCE hInst, const PTSTR pszTe
 	return hwndTip;
 }
 
+generic_string GetDialogItemString(HWND dialog, int item)
+{
+	constexpr int startSize = 256;
+	generic_string res(startSize, '\0');
+	auto stringSize = ::GetDlgItemText(dialog, item, &res[0], res.size());

```

Comment:
`res.size()` needs an explicit cast.
`std::wstring::size()` returns a `size_t`; on x86_64 machines, that's equal to `unsigned __int64`.

https://ci.appveyor.com/project/donho/notepad-plus-plus/builds/44667854/job/x8e9badc7uh0m344/messages

Reply:
Sorry about that.  Should be fixed now.